ArmPkg/ArmPkg.dsc

```diff
@@ -169,6 +169,7 @@
 
 [Components.common.MM_CORE_STANALONE]

```

Comment:
I'm not sure why this isn't caught by the parser, but I don't believe `Components.common.<MODULE_TYPE>` is valid. I don't know if the parser will ignore or still build. Can you remove the misspelled `MM_CORE_STANALONE` here and `MM_STANDALONE` below and just put these in the standard components section?

Reply:
Removed both section headers.